profile/Makefile

```diff
@@ -182,10 +183,10 @@ CFLAGS += -DVEC_ALIGNMENT=16
 # Optimization Flags
 ifeq ($(OPTIMIZE),yes)
 ifeq ($(COMPILER),gnu)
-  CFLAGS += -O3 -ffast-math -fpic
+  CFLAGS += -O3 -ffast-math -fpic -march=native
```

Comment:
this flag turns on architecture dependent optimizations. Since everyone is building OpenMOC from source it's fine. 
If the code was "clear" enough to the compiler, this also turns on SIMD vectorization

SIMD will be part of another PR

Reply:
Somewhat. The code can be fit for SIMD vectorization but not vectorized by the compiler, if the compiler doesn't realize it's legal to SIMD vectorize at that point